features/project/source/browse_files.feature

```diff
@@ -33,6 +33,13 @@ Feature: Project Source Browse Files
     And I click on "Commit Changes"
     Then I am redirected to the new file
     And I should see its new content
+    
+  @javascript
+  Scenario: I can upload file and commit
+    Given I click on "new file" link in repo
+    Then I can see new file page
+    And I can see "upload existing one"
+    And I click on "upload existing one"
```

Comment:
@stanhu I test it today without success, the Capybara is not able to find it. I attach one part of the debug source code.
![dropzone](https://cloud.githubusercontent.com/assets/7370334/9802801/432bc8d0-581d-11e5-8117-87e61dde2ee9.png)